R/derive_extreme_event.R

```diff
@@ -33,6 +33,8 @@
 #'   If a particular event from `events` has more than one observation, within
 #'   the event and by group, the records are ordered by the specified order.
 #'
+#'   `r roxygen_order_na_handling()`
```

Comment:
This is so cool!!! 
![image](https://github.com/pharmaverse/admiral/assets/10111024/d4badcd5-2c7a-4a50-ae2f-40568d2fd529)

Reply:
@pharmaverse/admiral This is a very cool use of the our `roxygen2.R` file with linking to a vignette for some extra guidance!!  Keep this one in your backpocket!